tests/test_data_processor.py

```diff
@@ -258,6 +258,30 @@ def test_wrong_extra_indexes_pandas(self):
         with self.assertRaises(ValueError):
             dp(df_raw)
 
+def test_norm_unnorm_preserves_coords_xr(self):
```

Comment:
Note for future reference that this is missing an indentation level @polpel - it should be part of the test class.